### DISCUSSION TITLE
How to use DP potential to predict lattice parameters?

### DISCUSSION THREAD

**Discussion by 13yong** [deepmodeling/deepmd-kit#1593] [Q&A]
Hello developer :
Recently I have read the artical : Deep learning of accurate force field of ferroelectric HfO2 ,written by Jing Wu and Shi Liu .(10.1103/PhysRevB.103.024108). In their work , they firstly trained a potential. And then they use it to predict lattice parameters. Through their hard work , the result whit DFT and DP , demonstrating excellent agreement. So I am very intering in this. Could you give me an idea if it's convenient?
The best wishes to you !

**Comment by michaelmacisaac** [comment]
There is an example script here:

https://icme.hpc.msstate.edu/mediawiki/index.php/LAMMPS_Tutorial_2.html
